docs/UserGuide.md

```diff
@@ -312,12 +383,12 @@ Action | Format, Examples
 **clear** | `clear`
 **delete** | `delete INDEX`<br> e.g., `delete 3`
 **edit** | `edit INDEX [n/NAME] [m/MODULE] [e/EMAIL] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
+**find** | `find [n/NAME] [m/MODULE] [t/TAG]...`<br> e.g., `find n/James Jake m/CS2100 t/T13`
```

Comment:
I believe find is able to do for multiple keyword

In fact, you could do ```find m/cs1231 m/cs2100``` or find "m/cs1231 cs2100". They will give you all students from cs1231 or cs2100